drivers/amazonec2/amazonec2.go

```diff
@@ -299,6 +299,12 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 			Usage:  "Encrypt the EBS volume using the AWS Managed CMK",
 			EnvVar: "AWS_ENCRYPT_EBS_VOLUME",
 		},
+		mcnflag.StringFlag{
+			Name:   "amazonec2-kms-key",
+			Usage:  "Custom KMS key using the AWS Managed CMK",
+			Value:  "alias/aws/ebs",
```

Comment:
don't set a default value, let the aws api do that. the param should come in as an empty string and we do not pass it into aws sdk and let them figure it out instead of us maintaining this "constant" that is the current default today.

Reply:
Changed the behavior of the flag.. if no custom key is specified a nil pointer is sent to the ec2.EbsBlockDevice, and AWS defaults to using the default encryption key `ebs/aws`.